scripts/install2

```diff
@@ -507,6 +508,26 @@ fi
 sudo cp status /usr/local/bin/status
 echo -e "Run \e[1;32mstatus\e[0m anytime to print the status of your node"
 
+# Setup nodeinfo.json, adding non-static info
+echo '"installed":'"$(date)"', "uri":"https://github.com/tomeshnet/prototype-cjdns-pi/tree/develop"},' >> shared/nodeinfo/nodeinfo.json
```

Comment:
When we eventually have a versioning system, that could go here along with the date. Also note that that uri will need to change when this gets merged to master.